evaluateReversePolishNotation.py

```diff
@@ -0,0 +1,22 @@
+class Solution(object):
+    def evalRPN(self, tokens):
+        """
+        :type tokens: List[str]
+        :rtype: int
+        """
+        stack = []
+        for char in tokens:
+            if char not in ('+', '-', '*', '/'):
```

Comment:
You can either do
```
if char not in '+-*/': 
```
for simplicity.
Or you can do
```
if char not in {'+', '-', '*', '/'}: 
```
for O(1) time complexity

Reply:
Sure, will do this